_posts/2022-05-16-changelog-170.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #170"
+date: 2022-05-16 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#7849] Add inspections and quick-fixes for `unused_must_use` and `clippy::double_must_use` lints
+
+  {% include gif-img.html path="/assets/posts/changelog-170/unused_must_use" w="700px" %}
```

Comment:
Let's take preview image from the corresponding gif. Otherwise, it leads to a weird rendering

<img width="811" alt="image" src="https://user-images.githubusercontent.com/2539310/168545948-ac74e2a2-b074-4a40-89fa-26a9972cafa9.png">